std/typecons.d

```diff
@@ -1212,6 +1217,8 @@ if (distinctFieldNames!(Specs))
          * $(TROW $(P `%(inner%|sep%)`), $(P The format `inner` is one format, that is applied
          *      on all fields of the `Tuple`. The inner format must be compatible to all
          *      of them.)))
+         *
+         * Example:
```

Comment:
As you already touch this code, it would be a good idea to move the examples from text into an actual, tested unittest block.

Reply:
Okay; that's trivial as the unittest block exists and the example is just duplication.